website/content/en/highlights/2023-11-07-0-34-0-upgrade-guide.md

```diff
@@ -31,3 +35,15 @@ been removed from the Enterprise configuration. Instead of `region`, `site` shou
 #### Removal of Deprecated `component_name` Metric Tag {#deprecated-component-name}
 
 The deprecated `component_name` tag has been removed from all internal metrics. Instead the `component_id` tag should be used.
+
+### Deprecations
+
+#### Deprecation of `requests_completed_total`, `request_duration_seconds`, and `requests_received_total` Internal Metrics {#deprcate-obsolete-http-metrics}
+
+The `requests_completed_total`, `request_duration_seconds`, and `requests_received_total` internal metrics will be removed in the 0.35.0 release.
```

Comment:
Will they be replaced with anything? Or we just don't think they are valuable?

Reply:
Yes, the replacements are provided in the table below.